### ISSUE TITLE
[FANBOX] 9999 Unsupported embed provider, please update content_provider.json for some post formats

### ISSUE THREAD

**Issue by AgentThirteen** [Nandaka/PixivUtil2#716]
### Prerequisites

* [x] Did you read FAQ section?
* [x] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?

### Description

SoundCloud (and possibly other video/sound formats) FANBOX posts will crash the script with the following error message:

```
9999 Unsupported embed provider = soundcloud for post = *, please update content_provider.json.
hasDumpPage=Y
pageData={'id': '*', 'title': '*', 'coverImageUrl': *, 'feeRequired': 0, 'publishedDatetime': '*', 'updatedDatetime': '*', 'type': 'video', 'body': {'text': '', 'video': {'serviceProvider': 'soundcloud', 'videoId': '*'}}, 'tags': [], 'excerpt': '', 'isLiked': *, 'likeCount': *, 'commentCount': *, 'restrictedFor': None, 'user': {'userId': '*', 'name': '*', 'iconUrl': '*'}, 'creatorId': '*', 'status': 'published'}
```

Workaround: Manually creating a skip post rule for such posts.

### Steps to Reproduce

1. Fetch any artist with a public or non-restricted post of that kind.
2. As soon as the script reaches an unsupported post format, fatals and exits.

**Expected behavior:** Downloading the post and attachment or skipping with a warning.

**Actual behavior:** Fatal.

### Versions
3a9b74cd558c6359c548a01d6f398d86f4ae8cb2

**Comment by Nandaka** [comment]
you can update the content_provider.json file and if possible upload it here.

**Comment by AgentThirteen** [comment]
[content_provider.json.txt](https://github.com/Nandaka/PixivUtil2/files/4674544/content_provider.json.txt)

Thanks for the pointers and very quick reply Nandaka.

I had to upload as txt because github wouldn't let me attach the JSON directly.

This should work fine when embedded in the HTML file with HTML file option on but I couldn't get the `contentId` anywhere from the API so I went with `videoId` alone.

The API flow now works properly but make sure you edit it to something less clunky if the content provider entry is weird.
